src/libNetwork/P2PComm.cpp

```diff
@@ -1020,19 +1053,32 @@ void P2PComm ::EventCbClientSeed([[gnu::unused]] struct bufferevent* bev,
     if (events & BEV_EVENT_EOF) {
       LOG_GENERAL(DEBUG, "P2PSeed BEV_EVENT_EOF");
     }
+    if (events & BEV_EVENT_TIMEOUT) {
+      LOG_GENERAL(DEBUG, "P2PSeed BEV_EVENT_TIMEOUT");
+    }
+    if (events & BEV_EVENT_CONNECTED) {
+      LOG_GENERAL(DEBUG, "P2PSeed BEV_EVENT_CONNECTED req msg len="
+                             << destBytes->size());
+    }
+  }
+  if (events & BEV_EVENT_CONNECTED) {
+    if (destBytes != NULL) {
+      if (bufferevent_write(bev, &(destBytes->at(0)), destBytes->size()) < 0) {
+        LOG_GENERAL(WARNING, "Error: P2PSeed bufferevent_write failed !!!");
+        return;
```

Comment:
For this case, which function deallocates `destBytes`?

Reply:
When we receive the response message from the server seed, `ReadCbClientSeed` would be called.After dispatching of message, we are calling `CloseAndFreeBevP2PSeedConnClient` and it will free the memory.
If there's no response then `BEV_EVENT_TIMEOUT` would be called.
Basically `CloseAndFreeBevP2PSeedConnClient` is the function where memory will be freed.